modules/op-release-notes-1-7.adoc

```diff
@@ -410,4 +410,14 @@ time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 ----
 +
 With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
```

Comment:
as a result, the S2I buildah and buildah tasks can run successfully

Reply:
or may be you can remove this line, as it is mentioned again in next line

Reply:
Resolved.